From b2d292de9a735c8b3af36ff6da48c9e7e3f6f6dd Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:45 -0400
Subject: [redhat] kabi: Adding symbol blk_start_queue (block/blk-core.c)

Message-id: <24f10cb25b0d660dd661454ec2a65ca1782e00f9.1410510773.git.poros@redhat.com>
Patchwork-id: 96064
O-Subject: [PATCH RHEL7.1 045/233] kabi: Adding symbol blk_start_queue (block/blk-core.c)
Bugzilla: 1062361
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Needed by PowerPath to restart a previously stopped queue.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_start_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_start_queue
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_start_queue

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_start_queue b/redhat/kabi/kabi-module/kabi_ppc64/blk_start_queue
new file mode 100644
index 0000000..7ddd250
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_start_queue
@@ -0,0 +1,2 @@
+#1-
+0xea346831 blk_start_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_start_queue b/redhat/kabi/kabi-module/kabi_s390x/blk_start_queue
new file mode 100644
index 0000000..474eb65
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_start_queue
@@ -0,0 +1,2 @@
+#1-
+0x66cd7232 blk_start_queue vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_start_queue b/redhat/kabi/kabi-module/kabi_x86_64/blk_start_queue
new file mode 100644
index 0000000..3a16c23
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_start_queue
@@ -0,0 +1,2 @@
+#1-
+0x149a3e2d blk_start_queue vmlinux EXPORT_SYMBOL
-- 
1.7.1